compiler/src/dmd/target.d

```diff
@@ -446,7 +446,7 @@ extern (C++) struct Target
             realsize = 10;
             realpad = 0;
             realalignsize = 2;
-            if (ptrsize == 4)
+            if (omfobj)
```

Comment:
This looks like a change for it's own pr.

Reply:
Well I would have had to disable some druntime tests temporarily on 32-bit Windows, because they test large structs and arrays. I don't remember which one(s).

Reply:
If you could split this off into a separate PR that would be much appreciated.